templates/patterns/file/pattern-file.html.twig

```diff
@@ -10,7 +10,6 @@
   'meta': '(' ~ file.size|format_size ~ ' - ' ~ file.extension|upper ~ ')',
   'icon': {
     'path': ecl_icon_path,
-    'type': 'general',
     'name': file.extension|to_file_icon,
```

Comment:
There is a problem here, the "presentation" icon used in the "to_file_icon" filter is not longer available.